packages/a11y/package.json

```diff
@@ -22,9 +22,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@leafygreen-ui/hooks": "^8.3.2",
-    "@leafygreen-ui/lib": "^14.0.0",
-    "@leafygreen-ui/emotion": "^4.0.8"
+    "@leafygreen-ui/hooks": "workspace:^",
+    "@leafygreen-ui/lib": "workspace:^",
+    "@leafygreen-ui/emotion": "workspace:^"
```

Comment:
Before we release I want to check that this does what we expect when we release (and document any potential process changes)